### DISCUSSION TITLE
how to change startup file?

### DISCUSSION THREAD

**Discussion by kai-studios** [vercel/community#3663] [Help] [answered]
### Summary

Hello! I am a new user of your hosting and uploaded my online chat to your hosting, but to start my chat you need to run the server.py file which is written in python. How can I do it?

### Example

https://middle-chat.vercel.app/

### Steps to Reproduce

Tell me how to reconfigure the startup file?

**Comment by amyegan** [comment, marked as answer]
Hey there. You can find more info about configuring Python Serverless Functions in the [Python Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python#) docs and in the [Python starter template](https://vercel.com/templates/python) repos.

**Reply by kai-studios** [reply]
Дякую!
